build/automation/var/project.mk

```diff
@@ -41,7 +41,7 @@ BLUE_GREEN_ENVIRONMENT := $(or $(BLUE_GREEN_ENVIRONMENT), $(ENVIRONMENT))
 SHARED_ENVIRONMENT := $(or $(SHARED_ENVIRONMENT), $(ENVIRONMENT))
 TF_VAR_blue_green_environment = $(BLUE_GREEN_ENVIRONMENT)
 TF_VAR_shared_environment = $(SHARED_ENVIRONMENT)
-
+AWS_PROFILE =
```

Comment:
What is the reason this needs to specified without a value ?

Reply:
This is to trick Terraform into not using the profile provider by the AWS config
`│ Error: error configuring Terraform AWS Provider: failed to get shared config profile, DI_SSO_MGMT
│
│   with provider["registry.terraform.io/hashicorp/aws"],
│   on common-provider.tf line 1, in provider "aws":
│    1: provider "aws" {`